README.md

```diff
@@ -1,104 +1,128 @@
+| ------------- |:-------------| :-----| :-----|
+| `gettingstarted` | feature | Simple example of how to read/write a Pravega `Stream`. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/gettingstarted)
+| `consolerw` | feature      | Application that allows users to work with `Stream`, `Transaction` and `StreamCut` APIs via CLI. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/consolerw)
+| `noop` | feature      | Example of how to add a simple callback executed upon a read event. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/noop)
+| `statesynchronizer` | feature | Application that allows users to work with `StateSynchronizer` API via CLI. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/statesynchronizer)
+| `streamcuts` | feature | Application examples demonstrating the use of `StreamCut`s via CLI. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/streamcuts) 
+| `turbineheatsensor` | scenario | It emulates parallel sensors producing temperature values (writers) and parallel consumers performing real-time statistics (readers). | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/turbineheatsensor)
+
+## Flink-connector Examples
+| Example Name        |  Type  | Description  | Language |
+| ------------- |:-------------| :-----| :-----|
+| `wordcount` | feature | Continuous count words on a Pravega `Stream` to demonstrate the usage of Flink connector. | [Java](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/java/io/pravega/examples/flink/wordcount)
+| `turbineheatprocessor` | scenario | A Flink streaming application for processing temperature data from a Pravega stream. Complements the `turbineheatsensor` app. The application computes a daily summary of the temperature range observed on that day by each sensor. | [Java](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/java/io/pravega/examples/flink/iot), [Scala](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/scala/io/pravega/examples/flink/iot)
+| `anomaly-detection` | scenario | A Flink streaming application for detecting anomalous input patterns using a finite-state machine. | [Java](https://github.com/pravega/pravega-samples/tree/master/anomaly-detection)
+
+# Build Instructions
+
+## Pre-requisites
+
+* Java 8
+
+* Maven 2
+
+## Pravega Build Instructions (Optional)
+
+If you have downloaded a nightly build of Pravega, you may need to generate the latest 
+Pravega `jar` files and publish them to your local Maven repository. For release builds of 
```

Comment:
Move the second line to title or make it the first line saying, Skip this step if you are using Pravega release artifacts from maven central repo.

Reply:
Done.